hack/Makefile.debug

```diff
@@ -18,13 +49,58 @@ push:
 set-image:
 	oc scale --replicas 0 -n openshift-cluster-version deployments/cluster-version-operator
 	oc scale --replicas 0 -n openshift-ingress-operator deployments ingress-operator
-	oc -n openshift-ingress scale deployment --replicas=0 router-default
-	oc -n openshift-ingress patch deployment router-default -p '{"spec":{"template":{"spec":{"$$setElementOrder/containers":[{"name":"router"}],"containers":[{"imagePullPolicy":"Always","name":"router"}]}}}}'
+	oc scale --replicas 0 -n openshift-ingress deployment/router-default
+	oc apply -f hack/remote-debug/manifests
+	@for patch in hack/remote-debug/patches/*.yaml; do \
```

Comment:
As discussed, since the router pod with the remote debugger needs all the capabilities and it needs to run as root, we have to add a patch for the privileged SCC. Example of the rbac for hostnework SCC: [link](https://github.com/openshift/cluster-ingress-operator/blob/192e2a62467d9fc2267d404214f6eaad3e724a91/pkg/manifests/assets/router/cluster-role.yaml#L46-L53).

Reply:
Done in 8b321249a352c4e7cced9d98e92680f00bff5867.